From f3f88020a301663048559b8245174f08650dff9c Mon Sep 17 00:00:00 2001
From: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
Date: Mon, 31 Oct 2016 15:06:36 -0700
Subject: [PATCH 1008/1008] Fixup libstdc++ header search paths for older
 versions of Mac OS X

The subpaths were removed in Lion.  Leopard and Snow Leopard had darwin8
symlinks for compatibility.

https://llvm.org/bugs/show_bug.cgi?id=30858

Signed-off-by: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
(cherry picked from commit e23e3e73fd7bc8e841725813d54df1ff5c0db88d)
---
 lib/Frontend/InitHeaderSearch.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git llvm_release_37/tools/clang/lib/Frontend/InitHeaderSearch.cpp macports_release_37/tools/clang/lib/Frontend/InitHeaderSearch.cpp
index e3a17c9..e08e615 100644
--- llvm_release_37/tools/clang/lib/Frontend/InitHeaderSearch.cpp
+++ macports_release_37/tools/clang/lib/Frontend/InitHeaderSearch.cpp
@@ -341,7 +341,7 @@ AddDefaultCPlusPlusIncludePaths(const llvm::Triple &triple, const HeaderSearchOp
                                   "powerpc-apple-darwin10", "", "ppc64",
                                   triple);
       AddGnuCPlusPlusIncludePaths("/usr/include/c++/4.0.0",
-                                  "powerpc-apple-darwin10", "", "ppc64",
+                                  "powerpc-apple-darwin8", "", "ppc64",
                                   triple);
       break;
 
@@ -350,7 +350,7 @@ AddDefaultCPlusPlusIncludePaths(const llvm::Triple &triple, const HeaderSearchOp
       AddGnuCPlusPlusIncludePaths("/usr/include/c++/4.2.1",
                                   "i686-apple-darwin10", "", "x86_64", triple);
       AddGnuCPlusPlusIncludePaths("/usr/include/c++/4.0.0",
-                                  "i686-apple-darwin8", "", "", triple);
+                                  "i686-apple-darwin8", "", "x86_64", triple);
       break;
 
     case llvm::Triple::arm:
-- 
2.10.1

